src/main/java/com/webonise/todoapp/service/UserService.java

```diff
@@ -0,0 +1,38 @@
+package com.webonise.todoapp.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.webonise.todoapp.Exception.PasswordDoesNotMatchException;
+import com.webonise.todoapp.Exception.UserExistsException;
+import com.webonise.todoapp.Exception.UserNotExistsException;
+import com.webonise.todoapp.dao.UserRepository;
+import com.webonise.todoapp.model.User;
+
+@Service
+public class UserService {
+	@Autowired
+	private UserRepository userRepository;
+	private Logger log = (Logger) LoggerFactory.getLogger(TodoService.class);
+
+	public Boolean verifyUser(User user) {
+		if (!userRepository.existsById(user.getUsername())) {
+			throw new UserNotExistsException();
+		} else {
+			if (userRepository.getOne(user.getUsername()).getPassword().equals(user.getPassword())) {
+				return true;
+			}
+		}
+		throw new PasswordDoesNotMatchException();
```

Comment:
Can you please clean this Exception should not be thrown by default.

Reply:
plus it is a security risk to expose which of the credential pair matches.It should throw only a single exception